src/Endpoints/Collections/Points.php

```diff
@@ -43,16 +45,18 @@ public function search(SearchRequest $searchParams): Response
     /**
      * @throws InvalidArgumentException
      */
-    public function scroll(Filter $filter = null): Response
+    public function scroll(Filter|ScrollRequest $scrollParams = null, array $queryParams = []): Response
     {
         $body = [];
-        if ($filter) {
-            $body['filter'] = $filter->toArray();
+        if ($scrollParams instanceof Filter) {
+            $body['filter'] = $scrollParams->toArray();
+        } elseif ($scrollParams instanceof ScrollRequest) {
+            $body = $scrollParams->toArray();
```

Comment:
I guess we can keep for now Filter for bc break. But at the end, we need to remove it. We can pass filter by using request too.